node/pegnet/winners.go

```diff
@@ -95,3 +96,62 @@ func (p *Pegnet) SelectMinerDominance(ctx context.Context, start, stop int) (Min
 
 	return result, nil
 }
+
+type GradedResult struct {
+	Height int32    `json:"height"`
+	Graded []Graded `json:"graded"`
+}
+
+type Graded struct {
+	EntryHash  string `json:"entryhash"`
+	Payout     uint64 `json:"payout"`
+	Nonce      string `json:"nonce"`
+	Difficulty string `json:"difficulty"`
```

Comment:
This should be a uint64 after decoding the blob in the db, not a hex string. The value in the db is always 8 bytes and you can decode it via: `Difficulty: binary.BigEndian.Uint64(difficulty)`